src/diagnosticfilename-inl.h

```diff
@@ -0,0 +1,54 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
We don't need this header in new files.

Reply:
If you are sure I will remove it, but just to confirm: The file is new, but the code in this file was copied from a file that had that header, so doesn't it need to keep the copyright statement?

Reply:
That's the approach we've been taking for some time now. My understanding is that we can't remove it from existing files, but new files don't need it.